version.cake

```diff
@@ -36,4 +38,78 @@ void UpdateSDKVersion(string path, string patternPrefix, string newVersion)
     ReplaceRegexInFiles(path, patternString, newString);
 }
 
+Task("UpdateCgManifestSHA").IsDependeeOf("StartNewVersion").Does(()=>
+{
+    try
+    {
+        var reposToUpdate = new List<string>(){ 
+            "https://github.com/microsoft/appcenter-sdk-android.git",
+            "https://github.com/microsoft/appcenter-sdk-apple.git",
+            "https://github.com/microsoft/appcenter-sdk-dotnet.git"
+        };
+        var manifestFilePath = "cgmanifest.json";
+        var gitHubApiPrefix = "https://api.github.com/repos";
+        var gitHubPrefix = "https://github.com";
+        var content = ParseJsonFromFile(manifestFilePath);
+        var registrations = (JArray)content["Registrations"];
+        foreach (var registration in registrations.Children())
+        {
+            var component = registration["component"];
+            if (component != null) 
+            {
+                var typeObject = component["type"];
+                if (typeObject != null && typeObject.Value<string>() == "git")
+                {
+                    var gitData = component["git"];
+                    var repoUrl = gitData["repositoryUrl"].Value<string>();
+                    if (reposToUpdate.IndexOf(repoUrl) >= 0)
+                    {
+                        var latestReleaseUrl = repoUrl.Replace(".git", "/releases/latest").Replace(gitHubPrefix, gitHubApiPrefix);
```

Comment:
Should be connected to this values instead of latest: https://github.com/microsoft/appcenter-sdk-unity/blob/340cd4ecb6d038d79919fc74965945bc0eaa30d9/build.cake#L16-L18

Reply:
Fixed